app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DisplayDatasetFragment.kt

```diff
@@ -103,9 +116,13 @@ class DisplayDatasetFragment : Fragment() {
 
         lifecycleScope.launch {
             dataset = dbManagement.getDatasetById(datasetId)!!
-            (binding.displayDatasetName as TextView).text = dataset.name
+            (_binding?.displayDatasetName as TextView).text = dataset.name
```

Comment:
This doesnt avoid the NPE
```suggestion
            (_binding?.displayDatasetName as? TextView)?.text = dataset.name
```

Reply:
you're totally right Thanks !